docs/GDI-SOP_ISM.md

```diff
@@ -0,0 +1,176 @@
+# European GDI - Procedures for Information Service Management for SOPs
+
+## Index
+
+1. [Document History](#1-document-history)
+2. [Glossary](#2-glossary)
+3. [Roles and Responsibilities](#3-roles-and-responsibilities)
+4. [Purpose](#4-purpose)
+5. [Scope](#5-scope)
+6. [Introduction and Background Information](#6-introduction-and-background-information)
+7. [Information Service Management Procedures](#7-information-service-management-procedures)
+   1. [GDI SOP creation](#71-gdi-sop-creation)
+      1. [European-level SOPs](#711-european-level-sops)
+      2. [Node-specific SOP templates](#712-node-specific-sop-templates)
+      3. [Node-specific SOPs](#713-node-specific-sops)
+   2. [GDI SOP accessioning and file naming](#72-gdi-sop-accessioning-and-file-naming)
+   3. [GDI SOP review and revision](#73-gdi-sop-review-and-revision)
+   4. [GDI SOP access rules](#74-gdi-sop-access-rules)
+8. [References](#references)
+
+## 1. Document History
+
+| Version | Author(s) | Description of Changes             | Date |
+| ------- | --------- | ---------------------------------- | ---- |
+| v0      |           | First version of document released |      |
+
+## 2. Glossary
+
+The following table defines the abbreviations and terms relevant to GDI SOPs.
+
+| Abbreviation | Description                            |
+| ------------ | -------------------------------------- |
+| OC           | Operations Committee                   |
+| SDPC         | Security and Data Protection Committee |
+
+| **Term**                     | **Definition**                                                                               |
+| ---------------------------- | -------------------------------------------------------------------------------------------- |
+| European-level SOP instances | Standard Operating Procedures (SOPs) to be followed for GDI processes at the European level. |
+| Node-specific SOP template   | Templates for SOPs that can be tailored to the specific processes of each node.              |
+
+## 3. Roles and Responsibilities
+
+| Role         | Full name             | GDI/node role   | Organisation            |
+| ------------ | --------------------- | --------------- | ----------------------- | 
+| **Author**   | Mallory Freeberg      | Task 4.3 member | EMBL-EBI                |
+| **Author**   | Marcos Casado Barbero | Task 4.3 member | EMBL-EBI                |
+| **Author**   | Silvia Bahena         | Task 4.3 member | EMBL-EBI                |
+| **Reviewer** | Jeroen Belien         | GDI-Member      | Health-RI/Amsterdam UMC |
+| **Reviewer** | Mattias Strömberg     | Task 4.3 member | UU                      |
+| **Reviewer** | Elisavet Torstensson  | Task 4.3 member | UU                      |
+| **Approver** |                       |                 |                         |     
+| **Approver** |                       |                 |                         | 
+
+## 4 Purpose

```

Comment:
I think it would be good to keep the numbering format consistent across the SOPs. The SOP0007 for examples uses 1., 2., 3. etc too and this would also go nicely with your sub-section numbering like 7.1.1 ...